src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs

```diff
@@ -169,6 +179,55 @@ public IActionResult PayeAornShutterPage([FromRoute] int ukprn)
         return View(PayeAornShutterPathViewPath, shutterViewModel);
     }
 
+    [HttpGet]
+    [Route("payeAornMatchedEmailNotLinked", Name = RouteNames.PayeAornMatchedEmailNotLinkedLink)]
+    public IActionResult PayeAornMatchedEmailNotLinkedShutterPage([FromRoute] int ukprn)
+    {
+        var sessionModel = _sessionService.Get<AddEmployerSessionModel>();
+        if (string.IsNullOrEmpty(sessionModel?.Paye))
+        {
+            return RedirectToRoute(RouteNames.AddEmployerStart, new { ukprn });
+        }
+
+        var cancelLink = Url.RouteUrl(RouteNames.AddEmployerStart, new { ukprn })!;
+        var shutterViewModel = new PayeAornMatchedEmailNotLinkedViewModel
+        {
+            EmployerName = sessionModel.OrganisationName!,
+            PayeReference = sessionModel.Paye!,
+            Aorn = sessionModel.Aorn!,
+            Email = sessionModel.Email,
+            CancelLink = cancelLink
+        };
+
+        return View(PayeAornMatchedEmailNotLinkedShutterPathViewPath, shutterViewModel);
+    }
+
+    [HttpPost]
+    [Route("payeAornMatchedEmailNotLinked", Name = RouteNames.PayeAornMatchedEmailNotLinkedLink)]
+    public IActionResult PostPayeAornMatchedEmailNotLinkedShutterPage([FromRoute] int ukprn)
+    {
+        var sessionModel = _sessionService.Get<AddEmployerSessionModel>();
+        if (string.IsNullOrEmpty(sessionModel?.Paye))
+        {
+            return RedirectToRoute(RouteNames.AddEmployerStart, new { ukprn });
+        }
+
+        if (sessionModel.PermissionToAddCohorts == null)
+        {
+            sessionModel.PermissionToAddCohorts = SetPermissions.AddRecords.Yes;
+            _sessionService.Set(sessionModel);
+        }
+
+        if (sessionModel.PermissionToRecruit == null)
+        {
+            sessionModel.PermissionToRecruit = SetPermissions.RecruitApprentices.Yes;
+            _sessionService.Set(sessionModel);
+        }
```

Comment:
This is already being taken care of on the `AddPermissionsAndEmployer` so it is redundant here. Also there is no need to have this POST action, instead the `Continue` can be directly linked to that page.